drakrun/drakrun/storage.py

```diff
@@ -307,6 +330,12 @@ def export_vm0(self, path: str):
     def import_vm0(self, path: str):
         shutil.copy(path, os.path.join(VOLUME_DIR, 'vm-0.img'))
 
+    def delete_vm_volume(self, vm_id: str):
+        # unmount can be done here
+        disk_path = os.path.join(VOLUME_DIR, f"vm-{vm_id}.img")
+        if safe_delete(disk_path) is False:
```

Comment:
AFAIK the idiomatic way would look like:
```suggestion
        if not safe_delete(disk_path):
```